dev-python/pyqtgraph/pyqtgraph-0.13.7.ebuild

```diff
@@ -23,19 +23,22 @@ S=${WORKDIR}/${PN}-${P}
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="amd64 ~arm arm64 ~riscv ~x86"
-IUSE="opengl svg"
-REQUIRED_USE="test? ( opengl svg )"
+IUSE="opengl svg pyside6 qt5 +qt6"
```

Comment:
I think we should consider naming the flags `pyside6 pyqt5 +pyqt6` instead, this I think reflects better what the flag actually does and then we also have symmetry with QtPy.ebuild.